api-reference/media-api/rename-file.md

```diff
@@ -2,7 +2,7 @@
 
 {% swagger baseUrl="https://api.imagekit.io" path="/v1/files/rename" method="put" summary="Rename file API" %}
 {% swagger-description %}
-You can programmatically rename an already existing file in the media library using rename file API.
+You can programmatically rename an already existing file in the media library using rename file API. This operation would rename all file versions of the file.
```

Comment:
We should also mention that the old url will stop working. And that files cached on CDN will continue to work unless a purge is requested.